src/main.rs

```diff
@@ -4,16 +4,76 @@ use cairo_foundry::cli::{
 	formatter::{self, Formatter},
 };
 use clap::Parser;
+use confy;
 use log::error;
+use serde_derive::{Deserialize, Serialize};
+
+const CONFIGURATION_FILENAME: &str = "config.toml";
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+struct Config {
+	args: String,
+}
+
+/// Tries to retrieve the Args from the CLI. If the user left the CLI empty, tries to
+/// retrieve the args from the configuration file.
+fn get_args() -> cli::Args {
+	// Try to parse arguments from stdin
+	let args_from_stdin: Result<cli::Args, clap::Error> = cli::Args::try_parse();
+
+	match args_from_stdin {
+		Err(stdin_err) => {
+			// Rust trick to help with type system
+			let stdin_err: clap::Error = stdin_err;
```

Comment:
Why is this necessary?
Can you explain in more details? Here or in the comment itself

Reply:
Not necessary but helps with `rust-analyzer` and other tools help understand what's going on :D

Reply:
6f1f899fa942ce8a8fbcd0d23466f33d89675d3f